### ISSUE TITLE
早报 @ 2019.05.06

### ISSUE THREAD

**Issue by sorrycc** [sorrycc/zaobao#133]
# 早报 @ 2019.05.06

## 发布

- **material-ui 发布 v4.0.0-beta.1**
  https://github.com/mui-org/material-ui/releases/tag/v4.0.0-beta.1
- **rollup 发布 v1.11.0**
  https://github.com/rollup/rollup/releases/tag/v1.11.0
- **preact 发布 10.0.0-beta.1 Los Compresseros**
  https://github.com/developit/preact/releases/tag/10.0.0-beta.1
- **chokidar 发布 3.0.0**
  CPU、RAM 和尺寸(17x)降低。 https://github.com/paulmillr/chokidar/releases/tag/3.0.0
- **docz 发布 v1.1.0**
  https://github.com/pedronauck/docz/releases/tag/v1.1.0
- **electron 发布 electron v6.0.0-beta.1**
  https://github.com/electron/electron/releases/tag/v6.0.0-beta.1
- **react-redux 发布 v7.1.0-alpha.4**
  听 dan 的建议，删除了 useActions
  https://github.com/reduxjs/react-redux/releases/tag/v7.1.0-alpha.4

## 文章

- **Fun with React Hooks - Michael Jackson and Ryan Florence - YouTube** `@tech("react-hooks")`
  https://www.youtube.com/watch?time_continue=7&amp;v=1jWS7cCuUXw
- **警惕拙劣的伪升维**
  https://mp.weixin.qq.com/s/WzXC_p7zUKE8KdXm5bMXTw
- **Introducing Serverless Pre-Rendering (SPR) - ZEIT**
  https://zeit.co/blog/serverless-pre-rendering
- **Docker For Frontend Developers**
  https://dev.to/akanksha_9560/docker-for-frontend-developers-1dk5
- **终于有神人把全国高铁图画得像坐地铁一样方便赶紧收藏了**
  https://mp.weixin.qq.com/s/HPtorp3frdWZyaLUXDNhfA
- **VS Code Remote 发布开启远程开发新时代**
  https://mp.weixin.qq.com/s/UF-thhDs96pV1_PqnkT0IQ
- **The tech behind my JavaScript education platform**
  https://gomakethings.com/the-tech-behind-my-javascript-education-platform/
- **Remote Development with Visual Studio Code** `@tech(editor)`
  https://code.visualstudio.com/blogs/2019/05/02/remote-development
- **SleepPhones Classic Headband Sleep Headphones With Cord**
  https://www.sleepsolutions.com.au/classic-sleepphones-headband-sleep-headphones
- **markdown-it 源码浅析 - 掘金**
  https://juejin.im/post/5cc92d1151882541303946f0?from=timeline&amp;isappinstalled=0
- **2019 年我们该如何重装 Windows 系统 | 实用技巧** `@tool`
  https://sspai.com/post/54281

## Star

- **insidegui/WWDC**
  The unofficial WWDC app for macOS
  https://github.com/insidegui/WWDC
- **agermanidis/autosub**
  Command-line utility for auto-generating subtitles for any video file
  https://github.com/agermanidis/autosub
- **iainreid820/drowsy**
  😴 Lazy integrations tool for RESTful interfaces to aid POC development and streamline integrations
  https://github.com/iainreid820/drowsy
- **Microsoft/TypeScript-New-Handbook**
  Incubation repository for the new TypeScript handbook 🐣
  https://github.com/Microsoft/TypeScript-New-Handbook
- **Microsoft/vscode-remote-release** `@tech(editor)`
  Visual Studio Code Remote Development: Open any folder in WSL, in a Docker container, or on a remote machine using SSH and take advantage of VS Code's full feature set.
  https://github.com/Microsoft/vscode-remote-release
- **zeit/vrs**
  A serverless virtual reality e-commerce experience powered by Now v2
  https://github.com/zeit/vrs
- **kahlil/zero-fux** `@tech(data-flow)`
  No Flux plus no Redux equals ZeroFux. A stateless unidirectional data flow implemented with Custom Events.
  https://github.com/kahlil/zero-fux
- **jamiebuilds/unstated-next** `@tech(data-flow)`
  200 bytes to never think about React state management libraries ever again
  https://github.com/jamiebuilds/unstated-next
- **csharad/rust-web-developer-roadmap** `@tech(rust)` `@resource`
  Roadmap to becoming a Rust Web Developer in 2019
  https://github.com/csharad/rust-web-developer-roadmap
- **vadimdemedes/pastel** `@tech(cli)`
  🎨 Framework for effortlessly building Ink apps
  https://github.com/vadimdemedes/pastel

## Tweets

- **I keep writing this useMount() hook.**
  It's easier to scan for in code than useEffect.
  Basically my brain can't reme… https://t.co/rnGQ5HFIaD
- **⚛️🛠 Following up on my thread yesterday about filter UI/UX- here's a mockup of how had I imagined the initial versi… https://t.co/GezGnJJ8mO**

---

往期早报：https://github.com/sorrycc/zaobao/issues ，想不漏收早报请点 watch 。

**Comment by neewbee** [comment]
请问这个 issue 是使用脚本生成的么？

**Comment by sorrycc** [comment]
@neewbee https://github.com/sorrycc/feed-to-taskpaper
